cdk/lib/appEnvironment.ts

```diff
@@ -1,15 +1,17 @@
-import { Secret as EcsSecret } from '@aws-cdk/aws-ecs';
-import { Secret } from '@aws-cdk/aws-secretsmanager';
-import { Construct } from '@aws-cdk/core';
+import {
+  aws_ecs as ecs,
+  aws_secretsmanager as secretsmanager,
+} from 'aws-cdk-lib';
+import { Construct } from 'constructs';
```

Comment:
This is typical of the import syntax changes for moving to v2. Things that were formerly in individual modules are now packaged as 'aws-ckd-lib', and the `Construct` class comes from a new 'constructs' package.

Reply:
Interesting. So by changing the names here, it makes it so you don't have to change code below?